src/meshes/p4est_mesh.jl

```diff
@@ -1427,13 +1444,18 @@ function balance!(mesh::P4estMesh{3}, init_fn=C_NULL)
   p8est_balance(mesh.p4est, P8EST_CONNECT_FACE, init_fn)
 end
 
+"""
+    p4est_partition_allow_for_coarsening!(mesh::P4estMesh, new_value::Bool)
```

Comment:
Are you sure we need the `p4est_` (maybe even the entire `p4est_partition_`) prefix? I am always for more descriptive names, but this seems overly long for a rather niche feature. Maybe something like `partition_with_coarsening_support`? Ah, I dunno 🤷

Reply:
The problem is that we do not turn off general coarsening with this :shrug:

Reply:
Alright, then unless @JoshuaLampert can come up with a better suggestion, feel free to close this...